src/classes/GE_GiftEntryController.cls

```diff
@@ -37,17 +37,21 @@
 */
 public with sharing class GE_GiftEntryController {
 
-    public static final Integer TIMEOUT_STATUS_CODE = 408;
-    public static final String TIMED_OUT = 'timed out';
     public static final String PURCHASE_CALL_TIMEOUT_MESSAGE = System.Label.geErrorRequestTimedout;
     private static final String BOOLEAN_DATA_TYPE = 'BOOLEAN';
 
-    @AuraEnabled(cacheable=true)
-    public static OrgDomainInfo getOrgDomainInfo() {
-        String url = System.Url.getOrgDomainUrl().getHost();
-        String[] urlParts = url.split('\\.');
-        String podName = [SELECT InstanceName FROM Organization].InstanceName.toLowerCase();
-        return new OrgDomainInfo (urlParts[0], podName);
+
+    /*******************************************************************************************************
+    * @description Indicates if the org is connected to Elevate
+    * @return Boolean
+    */
+    @AuraEnabled
```

Comment:
Could this be cachable?

Reply:
@ptewson-sfdo ^

Reply:
I believe so! The impact might be more on the testing side, where an update to the elevate parameters in the back-end might not immediately make the UI update until that cache is updated. I wouldn't mind making this function cacheable however.

Reply:
The method was set as `(cacheable=true)`.